lib/index.js

```diff
@@ -22,6 +23,7 @@ module.exports = {
         "playwright/no-focused-test": "error",
         "playwright/no-skipped-test": "warn",
         "playwright/no-wait-for-timeout": "warn",
+        "playwright/no-force-option": "warn",
```

Comment:
This one, I'm not sure about for the default rules, I can see both sides.  @mxschmitt Thoughts?